scripts/us_census/pep/annual_population/clean.py

```diff
@@ -0,0 +1,542 @@
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This Python Script contains methods to clean the county, state datasets
+"""
+import os
+import pandas as pd
+from state_to_geoid import USSTATE_MAP
+from fips_to_state import FIPSCODE
+
+
+def clean_df(df: pd.DataFrame, file_format: str) -> pd.DataFrame:
+    """
+    This Methods cleans the CSV dataset using the key Geographic Area as header
+    Arguments:
+        df (DataFrame) : Dataframe contains CSV data
+        file_format (str) : Sring value of either CSV, TXT
+    Returns:
+        df (DataFrame) : Cleaned Dataframe
+    """
+    if file_format.lower() == "csv":
+        header = "Geographic Area"
+        idx = df[df[0] == header].index
+        idx = idx.values[0]
+        df = df.iloc[idx:, :]
+        df = df.reset_index().drop(columns=["index"])
+    return df
+
+
+def find_file_format(path: str) -> str:
+    return path[path.rfind('.') + 1:]
+
+
+def _move_data_to_right(df: pd.DataFrame, row_index: list) -> pd.DataFrame:
+
+    for row in row_index:
+        for idx in range(7, 2, -1):
+            df.iloc[row, idx] = df.iloc[row, idx - 1]
+    return df
+
+
+def _get_nonna_index_for_tmp1(df: pd.DataFrame) -> pd.DataFrame:
+    return df[df["tmp1"].notnull()].index.values
+
+
+def _move_data_to_left(df: pd.DataFrame,
+                       row_values: list,
+                       idx1: int = -7,
+                       idx2: int = -2) -> pd.DataFrame:
+    for row in row_values:
+        for idx in range(idx1, idx2):  #-2:74, -3:73, -4:72, -5:71, -6:70
+            df.iloc[row, idx] = df.iloc[row, idx + 1]
+    return df
+
+
+def _get_numeric_index(df: pd.DataFrame, column: str) -> pd.DataFrame:
+    return df[df[column].apply(lambda row: row.isnumeric())].index.values
+
+
+def _get_char_index_in_col(df: pd.DataFrame, col: str) -> pd.DataFrame:
+    return df[df[col].str.replace(
+        ".", "").apply(lambda row: row.isalpha())].index.values
+
+
+def _merge_rows(df: pd.DataFrame, index_list: list, lr_idx: int, left_idx: int,
+                right_idx: int) -> pd.DataFrame:
+
+    for idx in index_list:
+        if lr_idx == 0:
+            df.iloc[idx - 1, -8:] = df.iloc[idx, left_idx:right_idx]
+        else:
+            df.iloc[idx - 1, -8:-1] = df.iloc[idx, left_idx:right_idx]
+        #print(df.iloc[idx-2:idx+2,:])
+    return df
+
+
+def _clean_county_df(county_df: pd.DataFrame,
+                     first_df_cols: list) -> pd.DataFrame:
+    """
```

Comment:
Pls add a line describing what this doesn, ie, expand 'clean'.

Function comments have a line with a brief description (more descriptive than the function name):
go/pystyle#function-docs